src/main/java/io/cdap/plugin/WindowsAggregationUtil.java

```diff
@@ -197,7 +197,7 @@ private static Schema schemaForInputField(Schema inputSchema, String inputFieldN
     return schemaField.getSchema();
   }
 
-  private static Column[] getPartitionsColumns(List<String> partitionsFields) {
+  public static Column[] getPartitionsColumns(List<String> partitionsFields) {
```

Comment:
I believe this is made public only for testing purposes. Please add a `@VisibleForTesting` annotation in that case.

Reply:
Done